src/SA/locale/entry.c

```diff
@@ -0,0 +1,36 @@
+#include <windows.h>
+#include "bofdefs.h"
+#include "base.c"
+#include "anticrash.c"
+
+DECLSPEC_IMPORT int WINAPI KERNEL32$GetLocaleInfoEx(LPCWSTR lpLocaleName, LCTYPE LCType, LPWSTR lpLCData, int cchData);
```

Comment:
Please move these into bofdefs.h

Reply:
I didn't originally as i felt like this would probably be the only place they would be used and I have previously opened tickets in the past when the bofdefs.h had too many in it and caused the BOF to fail.  I can certainly move it to there.

Reply:
if you're talking about the too many winapi calls error that Cobalt sometimes gives that's only raised if more then 32 winapi functions are actually called / referenced in the .c code.  Any number of these declarations can be in the .h and that in and of itself shouldn't cause that error.

I think the one we had in the past is someone updated base.c and added api calls in there without realizing it sent the number of referenced functions over 32 in some of our compiled .o files